kolibri/auth/api.py

```diff
@@ -175,7 +179,7 @@ def create(self, request):
         data = self.extract_request_data(request)
 
         # we validate the user's input, and if valid, login as user
-        serialized_user = FacilityUserSerializer(data=data)
+        serialized_user = self.serializer_class(data=data)
```

Comment:
Moved the class specification up to a class attribute, so it can be overridden here:
https://github.com/fle-internal/kolibri-instant-schools-plugin/pull/40/files#diff-f9beb280d2503af059c9651464593e62R26
(to keep the plugin as DRY as possible)